--- /usr/local/bin/jenkins.sh	2019-04-06 20:05:11.000000000 -0600
+++ /usr/local/bin/jenkins.sh	2019-04-06 20:30:27.000000000 -0600
@@ -1,5 +1,8 @@
 #! /bin/bash -e

+keystore_pass=`cat /var/nectr-keys/jenkins-keypass`
+JENKINS_OPTS="$JENKINS_OPTS --httpsKeyStorePassword=$keystore_pass"
+
 : "${JENKINS_WAR:="/usr/share/jenkins/jenkins.war"}"
 : "${JENKINS_HOME:="/var/jenkins_home"}"
 touch "${COPY_REFERENCE_FILE_LOG}" || { echo "Can not write to ${COPY_REFERENCE_FILE_LOG}. Wrong volume permissions?"; exit 1; }
